Fill out some TODOs in the code intel/code nav glossary

varungandhi-src opened the pull request:
- Fix broken links in code intel glossary
- Fill out some TODOs in the code intel/nav glossary

varungandhi-src committed:
Fix broken links in code intel glossary

varungandhi-src committed:
Fill out some TODOs in the code intel/nav glossary

scalabilitysolved commented on content/departments/engineering/teams/code-intelligence/glossary.md:
```diff
@@ -67,6 +107,16 @@ Examples:
+
+### definition
+
+TODO
```
Were you going to fill this in too or is it left as a future TODO?

varungandhi-src replied:
Future TODO.

scalabilitysolved reviewed (approved):
Looks good, some helpful additions 👍 

Left a single comment about a TODO that was added.

varungandhi-src commented:
@peterguy can you confirm if updated description in this PR is satisfactory or do you have any suggestions on things I should clarify?

peterguy committed:
tiny fix

"cannot be reliably by approximated by" --> "cannot be reliably approximated by"

peterguy commented:
Wow, @varungandhi-src, you rolled up your sleeves and dug into the whole glossary! Looks great!
